### DISCUSSION TITLE
I would like to star a star list

### DISCUSSION THREAD

**Discussion by loic-sharma** [community/community#8900] [Other Feature Feedback, Questions, & Ideas]
Imagine someone creates an awesome star list: https://github.com/stars/brianlovin/lists/personal-websites

I would like to be able to track this star list by starring it! Even better, I would like to create a star list of awesome star lists.

![image](https://user-images.githubusercontent.com/737941/146262454-8ed1d9ae-5c8e-40a8-b369-71a6195f91b5.png)

**Comment by brianlovin** [comment]
Yo dawg! Ha, love it — we have a tracking issue for this (and other ways to follow/consume List data and changes over time), so I'll add your +1 for our team to see 😊
